lib/benchpark/sys_repo/systems/tioga/system.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import pathlib
+
+from benchpark.system import System
+
+
+class Tioga(System):
+    def __init__(self):
+        super().__init__()
+
+        self.scheduler = "flux"
+        self.sys_cores_per_node = "64"
+        self.sys_gpus_per_node = "4"
+
+        base = pathlib.Path(os.path.abspath(__file__)).parents[0]
```

Comment:
Currently 

`./bin/benchpark system create --basedir=test-system-create --type=aws --set instance-type=c4.xlarge`

works, but not

`./bin/benchpark system create --basedir=test-system-create --type=tioga`

does not, because `__file__` is not available in this context: I'll need to figure out how to make that available.